doc/changelog/07-commands-and-options/12516-deprecate-grab-existentials.rst

```diff
@@ -0,0 +1,4 @@
+- **Deprecated:**
+  Grab Existential Variables and Existential commands
+  (`#12516 <https://github.com/coq/coq/pull/12516>`_,
+  by Pierre-Marie Pédrot).
```

Comment:
```suggestion
  by Maxime Dénès).
```

Reply:
Aha, the script is trying to attribute me work that is not mine!